docs/platform/release-notes/tizen-7-0-m1.md

```diff
@@ -0,0 +1,545 @@
+  - libaio
+    - Version 0.3.110 has been upgraded to version 0.3.112.
+  - libarchive
+    - Version 3.5.1 has been upgraded to version 3.5.2.
+  - libdatrie
+    - Version 0.2.10 has been upgraded to version 0.2.13.
+  - libevent
+    - Version 2.1.11 has been upgraded to version 2.1.12.
+  - libksba
+    - Version 1.3.0 has been upgraded to version 1.6.0.
+  - libpipeline
+    - Version 1.2.2 has been upgraded to version 1.5.0.
+  - libsigc++
+    - Version 2.9.3 has been upgraded to version 2.10.7.
+  - libsolv
+    - Version 0.7.16 has been upgraded to version 0.7.19.
+  - libtool
+    - Version 2.4.2 has been upgraded to version 2.4.6.
+  - libunistring
+    - Version 0.9.4 has been upgraded to version 0.9.10.
+  - libwbxml2
+    - Version 0.11.2 has been upgraded to version 0.11.7.
+  - libxml++
+    - Version 2.42.0 has been upgraded to version 3.2.3.
+  - libzip
+    - Version 1.7.3 has been upgraded to version 1.8.0.
+  - lsof
+    - Version 4.87 has been upgraded to version 4.91.
+  - m4
+    - Version 1.4.17 has been upgraded to version 1.4.19.
+  - make
+    - Version 4.0 has been upgraded to version 4.3.
+  - man-db
+    - Version 2.6.3 has been upgraded to version 2.9.4.
+  - meson
+    - Version 0.55.3 has been upgraded to version 0.60.3.
+  - mm-common
+    - Version 1.0.1 has been upgraded to version 1.0.3.
+  - mtools
+    - Version 4.0.18 has been upgraded to version 4.0.36.
+  - multipath-tools
+    - Version 0.4.9 has been upgraded to version 0.8.8.
+  - openfst
+    - Version 1.6.5 has been upgraded to version 1.8.1.
+  - paho-mqtt-c
+    - Version 1.3.7 has been upgraded to version 1.3.9.
+  - parted
+    - Version 3.1 has been upgraded to version 3.4.
+  - patch
+    - Version 2.7 has been upgraded to version 2.7.6.
+  - pcre
+    - Version 8.44 has been upgraded to version 8.45.
+  - perl
+    - Version 5.32.0 has been upgraded to version 5.34.0.
+  - pkg-config
+    - Version 0.28 has been upgraded to version 0.29.2.
+  - procps-ng
+    - Version 3.3.16 has been upgraded to version 3.3.17.
+  - psmisc
+    - Version 23.3 has been upgraded to version 23.4.
+  - python3
+    - Version 3.9.1 has been upgraded to version 3.9.10.
+  - re2
+    - Version 20201101 has been upgraded to version 20220201.
+  - re2c
+    - Version 1.1.1 has been upgraded to version 3.0.
+  - rpmlint
+    - Version 1.4 has been upgraded to version 1.11.
+  - rsync
+    - Version 3.1.1 has been upgraded to version 3.2.3.
+  - sgml-skel
+    - Version 0.6 has been upgraded to version 0.7.1.
+  - sqlite
+    - Version 3.33.0 has been upgraded to version 3.36.0.
+  - sudo
+    - Version 1.9.4 has been upgraded to version 1_9_7p2.
+  - tzdata
+    - Version 2020a has been upgraded to version 2021e.
+  - util-linux
+    - Version 2.36 has been upgraded to version 2.37.
+
+
+### Application framework
+
+#### New and changed features
+
+- Tizen boot sequence
+  - A new feature has been ready to control app launch on system boot.

```

Comment:
I believe implemented is the more appropriate word here.
```suggestion
  - A new feature has been implemented to control app launch on system boot.

```

Reply:
Will accept